docs/userguide/installation.md

```diff
@@ -32,6 +33,10 @@ your system. If it's not,
 [here](https://docs.anaconda.com/free/miniconda/index.html) is a really nice
 walkthrough for installing miniconda.
 
+Alternatively, you can install Python by downloading it from [python.org](https://www.python.org).
```

Comment:
Note that `which` is not available on powershell, so the above check will fail. Might want to rewrite this a little